nipype/algorithms/confounds.py

```diff
@@ -195,7 +195,8 @@ def _list_outputs(self):
 
 
 class FramewiseDisplacementInputSpec(BaseInterfaceInputSpec):
-    in_plots = File(exists=True, mandatory=True, desc='motion parameters as written by FSL MCFLIRT')
+    in_file = File(exists=True, mandatory=True, desc='motion parameters as written by FSL MCFLIRT or AFNI 3dvolreg')
+    format = traits.Enum("FSL", "AFNI", desc="Format of the motion parameters file: FSL (radians), AFNI (degrees)")
```

Comment:
why not use the same set of support that's in `ArtifactDetect`?

https://github.com/nipy/nipype/blob/86cd8ffb6dae1362a07ed8e9c79d583db87780eb/nipype/algorithms/rapidart.py#L163

and parameter interpretation is done here.

https://github.com/nipy/nipype/blob/86cd8ffb6dae1362a07ed8e9c79d583db87780eb/nipype/algorithms/rapidart.py#L42

ps. the composite norm output of artifact detect is another way of measuring motion.